src/helpers/gpt.ts

```diff
@@ -194,7 +254,7 @@ export const askGPT = async (question: string, chatHistory: CreateChatCompletion
     messages: chatHistory,
     model: "gpt-3.5-turbo-16k",
     max_tokens: config.review.tokenLimit,
-    temperature: 0,
+    temperature: 0.1,
```

Comment:
This should be `0` why did you change it?

Reply:
It seemed to help a lot with being less anal and pernickety about like white space in the gitignore and those silly requests

Reply:
I'm a bit skeptical about that but we'll roll with your current change.